docs/source/old_style.rst

```diff
@@ -0,0 +1,127 @@
+Old Style
```

Comment:
This can use a better name. What is `old` style? Is it what was used in `refactordoc`? If so, call it that.